partner_contact_vat/README.rst

```diff
@@ -0,0 +1,67 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=======================
+VAT in contact partners
+=======================
+
+This module was written to extend the functionality of contacts management to
+support setting a VAT in contact partners (those partners that are not not
+companies) and allow you to have separate VAT codes for contact and for their
+company.
+
+The company's VAT will still be used for invoiced and other legal documents.
```

Comment:
@Yajo please expand information on why we would want this as @dreispt said. When we will want to have the contact's VAT of a customer? Thanks